cocotb/share/lib/simulator/simulatormodule.c

```diff
@@ -203,7 +203,7 @@ static PyObject *log_msg(PyObject *self, PyObject *args)
 }
 
 
-// Register a callback for read only state of sim
+// Register a callback for ReadOnly state of sim
```

Comment:
ReadOnly() is the name of our trigger. The state of the simulation is actually read-only (or readonly).